styles/styles.css

```diff
@@ -364,6 +367,18 @@ main img {
   vertical-align: middle;
 }
 
+[class*=" vlt-icon-"], [class^="vlt-icon-"] {
```

Comment:
Adding an icon font family style so any decorations can be referenced by a simple class inclusion on the relevant element.

Reply:
Great idea. I was thinking about doing the same. Maybe we could have JS that parses the class name and generates the CSS to that adds the appropriate symbol to the `content`, e.g. `content: var(--vlt-phone-2-icon);`.

Reply:
Yup that would be an excellent addition!  I'll put in an issue (along with a couple of others that I need to collect) and mark them as improvements.